### ISSUE TITLE
Allow concatenating `Text`s that have `accessibilityIdentifier`s

### ISSUE THREAD

**Issue by sjavora** [kiwicom/orbit-swiftui#757]
In the same way that we concatenate Text with various visual modifiers (size, color), it would be great to also support accessibilityIdentifiers. This came up recently when only a part of a concatenated text was of interest for UI testing, but there was no way to add the identifier to only a part of the Text, e.g.:

```swift
Text("Cabin bag") + Text(count > 0 ? " \(count)x" : "")
```

And the identifier should only cover the second `Text` if possible.

**Comment by PavelHolec** [comment]
This cannot be done in native either. In those cases, the text needs to be split to two different texts, for example by HStack or other layout component.
